pkg/cloud/packet/client.go

```diff
@@ -38,6 +39,9 @@ const (
 	apiTokenVarName = "PACKET_API_KEY" //nolint:gosec
 	clientName      = "CAPP-v1beta1"
 	ipxeOS          = "custom_ipxe"
+	envVarLocalASN  = "METAL_LOCAL_ASN"
+	envVarBGPPass   = "METAL_BGP_PASS" //nolint:gosec
+	DefaultLocalASN = 65000
```

Comment:
Kube-vip currently relies on CPEM to enable BGP on the project and nodes. The method I used here to determine/configure these values are derived from CPEM. So I’m not sure if introspection would work here since then CPEM should also be able to introspect these values.

Reply:
So, I fully agree that with the current scenario we have a bootstrapping problem between CPEM and kube-vip, what I am suggesting is that we could work with the kube-vip project (or contribute the changes needed ourselves) in order to support enabling BGP configuration where needed rather than doing it here.

Reply:
I’d be happy to contribute the change to kube-vip. However, what I tried to get at is that it seems these values would need to be configured for kube-vip as well, as this is also the case for CPEM.

If I understand correctly, these values are later used by kube-vip to setup the BGP peering. So whatever enabled BGP on the project also needs to set the values, and there wouldn’t be a way to introspect them if BGP isn’t enabled on the project.

If what I’m saying doesn’t make sense I’m eager to understand why.

Reply:
I think the idea is that if you have kube-vip enable bgp on the project and the device, then the details like ASN and such can be pulled out via curl from the metadata server. 

https://metal.equinix.com/developers/docs/bgp/bgp-on-equinix-metal/

Reply:
@cprivite That's true, but then wouldn't the same process that I'm doing here just move to `kube-vip`? I guess the main question is if that fits within the scope of what `kube-vip` does or is expected to do.